<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>741</width>
    <height>573</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Stream Viewer</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icons.qrc">
    <normaloff>:/images/title.png</normaloff>:/images/title.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QTabBar::tab { 
width: 200px; height: 20px; 
 background: qlineargradient(spread:pad, x0:0, y1:1, x1:0, y1:0, stop:0 #bad7f9, stop:1 #a1c7f5);
border-right:1px solid #788aa3;
border-top:1px solid #788aa3;
border-left:1px solid #788aa3;
border-top-right-radius: 10px;
border-top-left-radius: 10px;
}

QTabBar::tab:selected {
width: 200px; height: 20px; 
 background: qlineargradient(spread:pad, x0:0, y1:1, x1:0, y1:0, stop:0 #ffffff, stop:1 #f7f8f8);
border-right:1px solid #788aa3;
border-top:1px solid #788aa3;
border-left:1px solid #788aa3;
border-top-right-radius: 6px;
border-top-left-radius: 6px;
}</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>741</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="newTabItem"/>
    <addaction name="separator"/>
    <addaction name="action"/>
   </widget>
   <widget class="QMenu" name="viewTopMenu">
    <property name="title">
     <string>Вид</string>
    </property>
    <addaction name="alwayOnTopItem"/>
    <addaction name="shortViewItem"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>Помощь</string>
    </property>
    <addaction name="actionOn_line"/>
    <addaction name="separator"/>
    <addaction name="aboutItem"/>
   </widget>
   <widget class="QMenu" name="menuBookmarks">
    <property name="title">
     <string>Bookmarks</string>
    </property>
   </widget>
   <addaction name="menu"/>
   <addaction name="viewTopMenu"/>
   <addaction name="menuBookmarks"/>
   <addaction name="menu_3"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="movable">
    <bool>true</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="alwayOnTopItem">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Поверх всех окон</string>
   </property>
  </action>
  <action name="action">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionOn_line">
   <property name="text">
    <string>On-line помощь</string>
   </property>
  </action>
  <action name="aboutItem">
   <property name="text">
    <string>О программе</string>
   </property>
  </action>
  <action name="newTabItem">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/images/tab-new.png</normaloff>:/images/tab-new.png</iconset>
   </property>
   <property name="text">
    <string>New Tab</string>
   </property>
   <property name="shortcut">
    <string/>
   </property>
  </action>
  <action name="shortViewItem">
   <property name="text">
    <string>Short View</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
